src/components/Settings/SettingsAddressbook.vue

```diff
@@ -171,19 +170,17 @@ export default {
 					this.$store.dispatch('toggleAddressbookEnabled', this.addressbook)
 				} catch (err) {
 					// error handling
+					console.error(err)
+					OC.Notification.showTemporary(t('contacts', 'Enabled toggle of addressbook was not successful.'))
 				} finally {
 					// stop loading status regardless of outcome
 					this.toggleEnabledLoading = false
 				}
 			}, 500)
 		},
 		downloadAddressbook() {
-			// change to loading status
-			this.downloadLoading = true
-			setTimeout(() => {
-				// stop loading status regardless of outcome
-				this.downloadLoading = false
-			}, 1500)
+			// Notify download started
+			OC.Notification.showTemporary(t('contacts', 'Downloading Addressbook'))
```

Comment:
Not needed, if the download starts, the user already knows it ;)

Reply:
ok will remove it :) anything else before merging?

Reply:
:see_no_evil: :grimacing: :rocket: GO